src/cljs/rems/administration/organizations.cljs

```diff
@@ -21,6 +22,11 @@
  ::fetch-organizations
  (fn [db]
    (let [description [text :t.administration/organizations]]
+     ;; Refresh :organizations subscription. We do this here
+     ;; since it would be surprising if an organization was visible on
+     ;; this page but not selectable e.g. when creating a resource.
+     (config/fetch-organizations!)
```

Comment:
I think this is a bit odd thing to do. It's not strictly necessary after the previous change?

Reply:
I think it's useful if e.g. you've created an org using the API and then want to use it via the UI, or perhaps somebody else just created an org and tells you to check it out.